test/test_models.py

```diff
@@ -149,7 +150,8 @@ def test_(case):
 
 @pytest.mark.endpoints()
 def test_response_from_requests(base_url):
-    response = requests.get(f"{base_url}/cookies")
+    headers = {"User-Agent": USER_AGENT}
```

Comment:
Could you please, add an explicit `assert` that the proper "User-Agent" header is present in `serialized.headers`?

Reply:
requests dont return User-Agent in response.

Reply:
Indeed, then this change is not relevant for this test